mobile/src/modules/auth/components/AccountListItem.tsx

```diff
@@ -83,7 +87,21 @@ export class AccountListItem extends React.PureComponent<Props> {
         backgroundColor='transparent'
       >
         <TouchableOpacity style={styles.view} onPress={this.handlePress}>
-          <View style={styles.col}>
+          <View
+            style={{
+              display: 'flex',
+              flexDirection: 'column',
+              flexWrap: 'wrap',
+              backgroundColor: AccountColors[accountIndex],
```

Comment:
not sure how to do dynamic styles or merge styles...

Reply:
I don't know if I understood you here, but merging styles can be done by ES spread operator